mlflow/server/__init__.py

```diff
@@ -30,6 +30,8 @@
 ARTIFACT_ROOT_ENV_VAR = "_MLFLOW_SERVER_ARTIFACT_ROOT"
 ARTIFACTS_DESTINATION_ENV_VAR = "_MLFLOW_SERVER_ARTIFACT_DESTINATION"
 PROMETHEUS_EXPORTER_ENV_VAR = "prometheus_multiproc_dir"
+ENABLE_STATISTICS_ENV_VAR = "_MLFLOW_ENABLE_STATISTICS"
```

Comment:
These can be migrated to ``mlflow/environment_variables.py`` and the access to them below can be handled via a ``.get()`` on the defaulted ``_BooleanEnvironmentVariable`` instance.

Reply:
Isn't it a CLI argument such as `PROMETHEUS_EXPORTER_ENV_VAR`?